great_expectations/expectations/core/expect_column_values_to_be_in_set.py

```diff
@@ -104,17 +105,62 @@ class ExpectColumnValuesToBeInSet(ColumnMapExpectation):
     }
 
     map_metric = "column_values.in_set"
+
+    args_keys = (
+        "column",
+        "value_set",
+    )
+
     success_keys = (
         "value_set",
         "mostly",
         "parse_strings_as_datetimes",
+        "auto",
+        "profiler_config",
     )
-    default_kwarg_values = {"value_set": None, "parse_strings_as_datetimes": False}
-    args_keys = (
-        "column",
-        "value_set",
+
+    default_profiler_config: RuleBasedProfilerConfig = RuleBasedProfilerConfig(
+        name="expect_column_values_to_be_in_set",  # Convention: use "expectation_type" as profiler name.
+        config_version=1.0,
+        variables={
+            "mostly": 1.0,
+        },
+        rules={
+            "default_column_values_between_rule": {
+                "domain_builder": {
+                    "class_name": "ColumnDomainBuilder",
+                    "module_name": "great_expectations.rule_based_profiler.domain_builder",
+                },
+                "parameter_builders": [
+                    {
+                        "name": "value_set",
+                        "class_name": "ValueSetMultiBatchParameterBuilder",
+                        "metric_domain_kwargs": "$domain.domain_kwargs",
+                    },
+                ],
+                "expectation_configuration_builders": [
+                    {
+                        "expectation_type": "expect_column_values_to_be_in_set",
+                        "class_name": "DefaultExpectationConfigurationBuilder",
+                        "column": "$domain.domain_kwargs.column",
+                        "value_set": "$parameter.value_set.value",
+                        "mostly": "$variables.mostly",
+                        "meta": {
+                            "profiler_details": "$parameter.value_set.details",
+                        },
+                    },
+                ],
+            },
+        },
     )
```

Comment:
Is this reusable at all between different expectations? I'd love to save us from having to write out full configs each time if possible but that may not be realistic.

Reply:
@cdkini My thinking on reusability is as follows: we create the (relatively simple) default profiler configurations for the individual expectations.  Then, it will be simple to refer them for various uses (in an a-la-carte fashion).  I have a notebook I can show you that is doing this (hat tip to @talagluck ).  Thanks!